tomcat/metadata.csv

```diff
@@ -11,7 +11,9 @@ tomcat.processing_time,gauge,10,,,The sum of request processing times across all
 tomcat.servlet.processing_time,gauge,10,,,The sum of request processing times across all requests to the servlet (in milliseconds) per second.,0,tomcat,servlet proc time
 tomcat.servlet.error_count,gauge,10,error,second,The number of erroneous requests received by the servlet per second.,-1,tomcat,servlet err count
 tomcat.servlet.request_count,gauge,10,request,second,The number of requests received by the servlet per second.,0,tomcat,servlet requests
-tomcat.cache.access_count,gauge,10,get,second,The number of accesses to the cache per second.,0,tomcat,cache accesses
-tomcat.cache.hits_count,gauge,10,hit,second,The number of cache hits per second.,0,tomcat,cache hits
+tomcat.string_cache.access_count,gauge,10,get,second,The number of accesses to the string cache per second.,0,tomcat,string cache accesses
+tomcat.string_cache.hit_count,gauge,10,hit,second,The number of string cache hits per second.,0,tomcat,string cache hits
+tomcat.web.cache.lookup_count,gauge,10,hit,second,The number of lookups for web resources per second.,0,tomcat,web cache lookups
+tomcat.web.cache.hit_count,gauge,10,get,second,The number of requests for web resources that were served from the cache per second.,0,tomcat,web cache accesses
```

Comment:
```suggestion
tomcat.web.cache.hit_count,gauge,10,hit,second,The number of requests for web resources that were served from the cache per second.,0,tomcat,web cache accesses
```

Reply:
Any reason for the description to be so different between the two caches? Other than that it seems that `lookup_count` is the amount of tries and `hit_count` is the number of successful reads so it seems like the unit should be switched between those two.